### ISSUE TITLE
 Generalize adding supported docker args

### ISSUE THREAD

**Issue by asbjornenge** [asbjornenge/cccf-docker-instructions#1]
YES I KNOW! :stuck_out_tongue_closed_eyes: 

We need a better way to add supported docker flags to all containers. Right now it's specific to each supported flag (env,dns). For this to be impl. right we need a **cccf-docker** module with a separate docker-cli-flags so that I can do:

```
var supportedArgs = require('cccf-docker').flags
```

Or something like that...

**Comment by asbjornenge** [comment]
Fuck it, I'm doing it NAO!

**Comment by asbjornenge** [comment]
Alright, so. I decided to NOT implement cccf-docker since it would be a regex pain and require lots of attention. The API, fields and values might change at any time and exists in many different versions.

So what I ended up doing is just allowing any additional argument (except --var which does variable replacement) to be reflected in the output. So if you go:

```
cdi run containers.json --env a=b --dns 172.17.42.1 --volume /tmp:/tmp
// =>
docker run --env a=b --dns 172.17.42.1 --volume /tmp:/tmp <image1> <cmd1>
docker run --env a=b --dns 172.17.42.1 --volume /tmp:/tmp <image2> <cmd2>
...
```

It will add those additional args passed to **cdi** to every **docker run** command.

I'm working on this on the [flexibility branch](../tree/flexibility). It will merge as soon as my [PR to tsiminim](https://github.com/Node-Ops/tsiminim/pull/1) is merged (or re-packaged).
